### ISSUE TITLE
Run SonoBus Headless?

### ISSUE THREAD

**Issue by carltesta** [sonosaurus/sonobus#109]
I'm wondering if it's possible to run SonoBus headless via the command line and automatically connect to a specific group.

**Comment by essej** [comment]
Yes, the current version of SonoBus actually does have a very basic headless mode (-q) where you'll need to specify a setup file on the
command line as well as a group to connect to. The idea is that you set up the config file ahead of time (using the GUI version) and save
the setup from there (audio options, etc). Then start it up on your headless system using the command line options. See all the options with -h.

**Comment by YuriVivaldi** [comment]
Is there a guide on the setup file syntax?
I'm trying to use multiple headless instances on embedded, so I need to pair each instance with a ALSA output card, in some way

I looked in the project for some docs, but I didn't find them

**Comment by essej** [comment]
It’s not really meant to be hand written… if you can run the standalone app with a UI on the system (or one just like it), you would set up all the audio (and other) options then use the Save Setup command from the first menu. Then you would specify the path to that setup file on the command line when running it in headless mode.

**Comment by YuriVivaldi** [comment]
So it's not possible to dynamically add headless clients?
Thats my goal, to create a network of listeners and to be able to configure them programmatically from a third manager/app, which I could build knowing the configuration syntax

**Comment by soldadocoder** [comment]
Has anyone had success with SonoBus in Headless mode?

I couldn't figure out where to place the (-q) command.
